Explorer/Assets/DCL/Backpack/BackpackBus/BackpackCommand.cs

```diff
@@ -19,10 +19,12 @@ public BackpackEmoteSlotSelectCommand(int slot)
     {
         public readonly string Id;
         public readonly int? Slot;
+        public readonly bool IsInitialEquip;
 
-        public BackpackEquipEmoteCommand(string id, int? slot = null)
+        public BackpackEquipEmoteCommand(string id, int? slot = null, bool isInitialEquip = false)
```

Comment:
Is not clean to me to pass this parameter in many places. Instead, you could simplify by doing some equality comparison with the previous and the new profile that is going to be published. In case they are the same, skip it. What do you think?